### PR TITLE
[WIP] Added anlysis_work.txt + migrate script for mrp.bom product_eff…

### PR DESCRIPTION
**Pull request by PCatinean**
Work in progress

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/migrations/10.0.2.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
```
Comment (NL66278) [inline addons/mrp/migrations/10.0.2.0/pre-migration.py]: If nothing is done in pre-migrate.py, it should not be included.

**Review by NL66278 — APPROVED** [review]
:+1: Mostly approve but leave out unused script

**Comment by pedrobaeza** [comment]
Please rebase over current branch. Is this finished?

**Comment by PCatinean** [comment]
@pedrobaeza rebased and removed pre-migrate.py. Hardly 'finished' since it's just been analysis but yes this is as far as I got given my time and knowledge about OpenUpgrade and MRP :(

**Comment by pedrobaeza** [comment]
OK, let's see if anyone more wants to continue this.

**Comment by MiquelRForgeFlow** [comment]
I would like to continue. How I proceed to?

**Comment by pedrobaeza** [comment]
Take this branch as base and open a new PR with your additions.

**Comment by pedrobaeza** [comment]
Superseeded by #1076

**Pull request closed without merging**